package.json

```diff
@@ -82,33 +82,33 @@
     "@types/storybook__react": "^5.2.1",
     "@types/styled-components": "^5.1.7",
     "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^10.1.0",
+    "babel-eslint": "^10.0.1",
     "babel-jest": "^24.8.0",
     "babel-loader": "^8.2.2",
     "babel-plugin-styled-components": "^1.12.0",
     "babel-plugin-transform-imports": "^2.0.0",
     "bundlesize": "^0.18.0",
-    "chromatic": "^5.6.0",
+    "chromatic": "^5.0.0",
     "clean-webpack-plugin": "^3.0.0",
     "copy-webpack-plugin": "^5.0.4",
-    "copyfiles": "^2.4.1",
-    "core-js": "^3.8.3",
+    "copyfiles": "^2.3.0",
+    "core-js": "^3.6.4",
     "cross-env": "^5.1.6",
     "del": "^5.1.0",
-    "eslint": "^7.18.0",
-    "eslint-config-airbnb": "^18.2.1",
-    "eslint-config-prettier": "^7.2.0",
-    "eslint-plugin-babel": "^5.3.1",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-prettier": "^3.3.1",
-    "eslint-plugin-react": "^7.22.0",
+    "eslint": "^6.2.2",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-config-prettier": "^6.1.0",
+    "eslint-plugin-babel": "^5.1.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^6.2.1",
+    "eslint-plugin-prettier": "^3.0.0",
+    "eslint-plugin-react": "^7.9.1",
     "eslint-plugin-react-hooks": "^2.2.0",
     "fs-extra": "^8.1.0",
     "grommet-theme-aruba": "^0.1.1",
     "grommet-theme-dxc": "^0.1.1",
     "grommet-theme-hp": "^0.1.1",
-    "grommet-theme-hpe": "^2.0.2",
+    "grommet-theme-hpe": "^2.0.0",
```

Comment:
The `package.json` changes look like a result of a bad merge conflict, please be mindful it downgrades the mentioned dependencies.

Reply:
`package.json` is inline with master and diffs are no longer present.